### PR TITLE
Add release notes draft for 13.4

### PR DESCRIPTION
**Pull request by oliverguenther**

### CODE REVIEW

File: docs/release-notes/13-4-0/openproject-13-4-nudge-oauth.png
Comment (akabiru) [inline docs/release-notes/13-4-0/openproject-13-4-nudge-oauth.png:1]: 👋🏾 Hi, we might want to update to a more recent screenshot; the design was updated

Screenshot 2024-03-11 at 3 29 45 PM

File: docs/release-notes/13-4-0/README.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+### GitLab integration
+
+We are delighted about the GitLab integration, which is officially available with OpenProject 13.4. It goes back to the plugin that Community contributor [Benjamin Tey](https://community.openproject.org/users/69094) created 3 years ago. Many thanks at this point to everyone who worked on the GitLab integration and the plugin – inside and outside of OpenProject's core development team! Such projects are a great example of the power of open source collaboration.💙
+
+With this integration, you can manage GitLab's units of work (merge requests and issues) from within OpenProject by linking them with work packages. Read more about OpenProject's GitLab integration [in our system admin guide](https://www.openproject.com/docs/system-admin-guide/integrations/gitlab-integration/).
+
+<div class="blog-box-info">	
+
+**Already using the user-generated GitLab plugin?** With OpenProject 13.4, the existing plugin will be replaced by the new integration. We recommend removing the plugin and replacing it with the GitLab integration.
```
Comment (akabiru) [inline docs/release-notes/13-4-0/README.md]: From https://github.com/btey/openproject-gitlab-integration/pull/60

```suggestion
**Already using the user-generated GitLab plugin?** With OpenProject 13.4, the existing plugin will be replaced by the new integration. We recommend removing the plugin module folder and bundler references before upgrading to openproject. Your historical dataset will remain unaffected within OpenProject as there were no changes to the data model.

Before upgrading, please do the following:

1. Remove traces of the gitlab integration in your **Gemfile.lock** and **Gemfile.modules** See [btey/openproject-gitlab-integration#configuration](https://github.com/btey/openproject-gitlab-integration?tab=readme-ov-file#configuration)
2. Remove the module code traces of the gitlab integration. `rm -rf /path/to/openproject/modules/gitlab_integration` 
```

File: docs/release-notes/13-4-0/README.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+With this integration, you can manage GitLab's units of work (merge requests and issues) from within OpenProject by linking them with work packages. Read more about OpenProject's GitLab integration [in our system admin guide](https://www.openproject.com/docs/system-admin-guide/integrations/gitlab-integration/).
+
+<div class="blog-box-info">	
+
+**Already using the user-generated GitLab plugin?** With OpenProject 13.4, the existing plugin will be replaced by the new integration. We recommend removing the plugin module folder and bundler references before upgrading to OpenProject. Your historical dataset will remain unaffected within OpenProject as there were no changes to the data model.
+
+Before upgrading, please do the following:
+
+1. Remove traces of the GitLab integration in your **Gemfile.lock** and **Gemfile.modules**. See [btey/openproject-gitlab-integration#configuration](https://github.com/btey/openproject-gitlab-integration?tab=readme-ov-file#configuration)
```
Comment (akabiru) [inline docs/release-notes/13-4-0/README.md]:
```suggestion
1. Remove traces of the GitLab integration in your **Gemfile.lock** and **Gemfile.modules**. See [btey/openproject-gitlab-integration#configuration](https://github.com/btey/openproject-gitlab-integration?tab=readme-ov-file#configuration) Failure to do so may result in a `Bundler::GemfileError` matching the following error message: _Your Gemfile lists the gem openproject-gitlab_integration (>= 0) more than once._
```

**Review by MayaBerd — COMMENTED** [review]
I've merged small wording changes directly

File: docs/release-notes/13-4-0/README.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+With this integration, you can manage GitLab's units of work (merge requests and issues) from within OpenProject by linking them with work packages. Read more about OpenProject's GitLab integration [in our system admin guide](https://www.openproject.com/docs/system-admin-guide/integrations/gitlab-integration/).
+
+<div class="blog-box-info">	
+
+**Already using the user-generated GitLab plugin?** With OpenProject 13.4, the existing plugin will be replaced by the new integration. We recommend removing the plugin module folder and bundler references before upgrading to OpenProject. Your historical dataset will remain unaffected within OpenProject as there were no changes to the data model.
+
+Before upgrading, please do the following:
+
+1. Remove traces of the GitLab integration in your **Gemfile.lock** and **Gemfile.modules**. See [btey/openproject-gitlab-integration#configuration](https://github.com/btey/openproject-gitlab-integration?tab=readme-ov-file#configuration) Failure to do so may result in a `Bundler::GemfileError` matching the following error message: _Your Gemfile lists the gem openproject-gitlab_integration (>= 0) more than once._
```
Comment (MayaBerd) [inline docs/release-notes/13-4-0/README.md]:
```suggestion
1. Remove traces of the GitLab integration in your **Gemfile.lock** and **Gemfile.modules**. See [btey/openproject-gitlab-integration#configuration](https://github.com/btey/openproject-gitlab-integration?tab=readme-ov-file#configuration). Failure to do so may result in a `Bundler::GemfileError` matching the following error message: _Your Gemfile lists the gem openproject-gitlab_integration (>= 0) more than once._
```

**Pull request merged**